### ISSUE TITLE
Unused imports ignored

### ISSUE THREAD

**Issue by tonoslav** [palantir/tslint#3425]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.5.3
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
...
import * as PropTypes from 'prop-types'
...
```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint-react"
    ],
    "jsRules": {},
    "rules": {
        "no-unused-variable": true,
        "no-unused-imports": true,
        "jsx-no-multiline-js": false,
        "jsx-boolean-value": false,
        "prefer-for-of": false,
        "ordered-imports": false,
        "no-angle-bracket-type-assertion": false,
        "object-literal-sort-keys": false,
        "member-ordering": false,
        "arrow-parens": false,
        "max-line-length": {
            "options": [5000]
        },
        "new-parens": true,
        "no-arg": true,
        "no-bitwise": true,
        "no-conditional-assignment": true,
        "no-consecutive-blank-lines": false,
        "no-console": {
            "severity": "warning",
            "options": [
                "debug",
                "info",
                "log",
                "time",
                "timeEnd",
                "trace"
            ]
        }
    },
    "rulesDirectory": []
}
```

#### Actual behavior
**PropTypes** is not used in file so I should get some error
![image](https://user-images.githubusercontent.com/3399021/32218303-c02ed1e0-be2a-11e7-9d92-2c84986754b8.png)

#### Expected behavior
No error

**Comment by ajafff** [comment]
`no-unused-imports` is not a tslint core rule.
However `no-unused-variable` should find them as well.
The problem is that the vscode extension does not run tslint with type information. Therefore you don't see failures of rules that need the type checker. When running `tslint -p. ` from CLI you will see the error as expected

**Comment by ajafff** [comment]
You can't due to technical limitations.
But you can try https://marketplace.visualstudio.com/items?itemName=eg2.ts-tslint
